core/state/dump.go

```diff
@@ -110,9 +110,7 @@ func (d iterativeDump) OnAccount(addr common.Address, account DumpAccount) {
 		SecureKey: account.SecureKey,
 		Address:   nil,
 	}
-	if addr != (common.Address{}) {
-		dumpAccount.Address = &addr
-	}
+	dumpAccount.Address = &addr
```

Comment:
Fixed, ptal

Reply:
I understand the fix, but this will break `t8n`: https://github.com/ethereum/go-ethereum/tree/master/cmd/evm: https://github.com/ethereum/go-ethereum/blob/5b792e0fdfdb71238f160d58553d87a2bfef3891/cmd/evm/internal/t8ntool/transition.go#L303

Reply:
Fixed to make evmtools happy.